Example/Example/UnknownEditorViewController.swift

```diff
@@ -11,14 +11,14 @@ class UnknownEditorViewController: UIViewController {
     ///
     fileprivate(set) var saveButton: UIBarButtonItem = {
         let saveTitle = NSLocalizedString("Save", comment: "Save Action")
-        return UIBarButtonItem(title: saveTitle, style: .plain, target: self, action: #selector(saveWasPressed))
+        return UIBarButtonItem(title: saveTitle, style: .plain, target: UnknownEditorViewController.self, action: #selector(saveWasPressed))
```

Comment:
This doesn't seem right 🤔 
It's now setting a class as a target.

Reply:
Good catch! These are Xcode's code suggestions, which I blindly took... Addressed in https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1400/commits/fc8861a90e17bc9faeab205f3d0faa457ee3e5ee.